package.json

```diff
@@ -3,15 +3,15 @@
   "description": "Reaction is a modern reactive, real-time event driven ecommerce platform.",
   "main": "index.js",
   "engines": {
-    "node": ">=14.18.1",
+    "node": ">=18.10.0",
     "npm": ">=7"
   },
   "engineStrict": true,
   "scripts": {
     "start:dev": "npm run start:dev -w apps/reaction",
     "start:meteor-blaze-app": "npm run start -w=apps/meteor-blaze-app",
     "build:packages": "pnpm -r run build",
-    "test": "pnpm -r run test",
+    "test": "pnpm -r run test --forceExit",
```

Comment:
Is this the right place to put this parameter?

Reply:
I believe the parameter was correctly passed to jest as we could see in the earlier successful unit-tests (ex test-unit (79061)).

But thinking about it now, this was one of the option suggested when we had the unit-test stalling with jest v25. Now after the upgrade, the tests are passing without this param. Hence I have removed the same and all tests passed.